src/EditorFeatures/Core/Shared/Tagging/Utilities/TagSpanIntervalTree.cs

```diff
@@ -74,13 +80,19 @@ public bool IsEmpty()
 
         public IEnumerable<ITagSpan<TTag>> GetIntersectingTagSpans(NormalizedSnapshotSpanCollection requestedSpans)
```

Comment:
ideally, these bridge methods woudl go away as well.  but there are still callers and i was trying to keep this PR under control.

Reply:
Looks like for this method there might only be one caller left (CopyPasteAndPrintingClassificationBufferTaggerProvider), so it might be pretty trivial to remove

Reply:
Ok. I'll see what I can do :-)

Reply:
ok.  removed :)